### PR TITLE
Enhance Aad samples to keep working correctly. 

### PR DESCRIPTION
**Pull request by Netyyyy**
1. Change some confusing descriptions and delete the unused descriptions or codes.
2. Add PowerShell way to use Terraform.

### CODE REVIEW

File: aad/spring-cloud-azure-starter-active-directory/web-client-access-resource-server/aad-resource-server-obo/README.md [hunk 1]
```diff
@@ -85,38 +80,29 @@ Graph response success.
 - Web API A will call Graph resource through `OAuth2AuthorizedClientRepository`. 
 
 ```shell script
+# same access-token with above
```
Comment (jialigit) [inline aad/spring-cloud-azure-starter-active-directory/web-client-access-resource-server/aad-resource-server-obo/README.md:83]: same access-token as above

File: aad/spring-cloud-azure-starter-active-directory-b2c/aad-b2c-resource-server/README.md [hunk 2]
```diff
@@ -101,19 +101,22 @@ mvn spring-boot:run
...
-```
-```http request
-GET /log HTTP/1.1
-Authorization: Bearer eyJ0eXAiO ... 0X2tnSQLEANnSPHY0gKcgw
+# get access_token
+curl -H "Content-Type: application/x-www-form-urlencoded" -d 'grant_type=client_credentials&client_id=<aad-b2c-web-app-client-id>&scope=<app-id-uri>/.default&client_secret=<aad-b2c-web-app-client-secret>' 'https://login.microsoftonline.com/<tenant-id>/oauth2/v2.0/token'
+
+# GET /write HTTP/1.1
+curl localhost:8080/write -H "Authorization: Bearer <access_token>"
+Write success
```
Comment (backwind1233) [inline aad/spring-cloud-azure-starter-active-directory-b2c/aad-b2c-resource-server/README.md:111]: Is "Write success" is the response?

Comment (Netyyyy) [reply aad/spring-cloud-azure-starter-active-directory-b2c/aad-b2c-resource-server/README.md:111]: Yes，it is.

File: aad/spring-cloud-azure-starter-active-directory/aad-resource-server-by-filter/README.md [hunk 3]
```diff
@@ -84,7 +84,9 @@ As a first step you'll need to:
    Click the **Grant/revoke admin consent for {tenant}** button, and then select **Yes** when you are asked if you want to grant consent for the
    requested permissions for all account in the tenant. ![grant admin consent](docs/grant-admin-consent.png "grant admin consent")
    You need to be an Azure AD tenant admin to do this.
-   
+
+1. Then back to **Azure Active Directory**, In the left-hand navigation pane, select **Groups**, and then select `group1` which you created, choose your tenant as **Owners** and Spring Boot Sample as **Members**.
```
Comment (chenrujun) [inline aad/spring-cloud-azure-starter-active-directory/aad-resource-server-by-filter/README.md:88]: `In the left-hand navigation pane` -> `in the left-hand navigation pane`.

Comment (chenrujun) [reply aad/spring-cloud-azure-starter-active-directory/aad-resource-server-by-filter/README.md:88]: > and then select `group1` which you created, choose your tenant as **Owners** and Spring Boot Sample as **Members**.

Is this necessary? IMU, the only thing we need to do is set `user` as member of `group1`.

File: aad/spring-cloud-azure-starter-active-directory/web-client-access-resource-server/aad-resource-server-obo/README.md [hunk 4]
```diff
@@ -74,6 +68,7 @@ mvn spring-boot:run
 First, you need to obtain an access token to access Web API A.
 - Web API A will call Graph resource. 
 ```shell script
+# use Header scope '<app-id-uri>/Obo.Graph.Read' to get access-token
```
Comment (chenrujun) [inline aad/spring-cloud-azure-starter-active-directory/web-client-access-resource-server/aad-resource-server-obo/README.md:71]: `use` -> `Use`.

File: aad/spring-cloud-azure-starter-active-directory/web-client-access-resource-server/aad-resource-server-obo/README.md [hunk 5]
```diff
@@ -85,38 +80,29 @@ Graph response success.
 - Web API A will call Graph resource through `OAuth2AuthorizedClientRepository`. 
 
 ```shell script
+# same access-token as above
```
Comment (chenrujun) [inline aad/spring-cloud-azure-starter-active-directory/web-client-access-resource-server/aad-resource-server-obo/README.md:83]: `same` -> `Same`.

**Review by backwind1233 — APPROVED** [review]
LGTM

File: aad/spring-cloud-azure-starter-active-directory-b2c/aad-b2c-resource-server/README.md [hunk 6]
```diff
@@ -101,19 +101,22 @@ mvn spring-boot:run
 ```
 
 ### Access the Web API
-We could use Postman to simulate a Web APP to send a request to a Web API.
 
 ```http request
```
Comment (chenrujun) [inline aad/spring-cloud-azure-starter-active-directory-b2c/aad-b2c-resource-server/README.md:105]: `http request` cannot be recognized in my side.

How about doing like this:

image

File: aad/spring-cloud-azure-starter-active-directory/web-client-access-resource-server/aad-resource-server-obo/README.md [hunk 7]
```diff
@@ -85,38 +80,29 @@ Graph response success.
...
 # Replace to valid access token.
 curl localhost:8081/call-graph-with-repository -H "Authorization: Bearer <replace-the-access-token>"
 ```
 
 Verify response:
 ```text
-Graph response success.
+Graph response failed.
```
Comment (chenrujun) [inline aad/spring-cloud-azure-starter-active-directory/web-client-access-resource-server/aad-resource-server-obo/README.md:90]: Why it's failed?

File: aad/spring-cloud-azure-starter-active-directory/web-client-access-resource-server/aad-resource-server-obo/README.md [hunk 8]
```diff
@@ -85,38 +80,29 @@ Graph response success.
...
 ```
 
 - Web API A will call Custom(Web API B) resources. 
 
 ```shell script
+# use Header scope '<app-id-uri>/Obo.WebApiA.ExampleScope' to get access-token
```
Comment (chenrujun) [inline aad/spring-cloud-azure-starter-active-directory/web-client-access-resource-server/aad-resource-server-obo/README.md:96]: `use` -> `Use`.

How about moving `# use ...` out of `shell script`?

File: aad/spring-cloud-azure-starter-active-directory/web-client-access-resource-server/aad-resource-server-obo/README.md (hunk 7 above)
Comment (Netyyyy) [reply aad/spring-cloud-azure-starter-active-directory/web-client-access-resource-server/aad-resource-server-obo/README.md:90]: It's just a output

**Pull request merged**